diff --git a/media/ffvpx/libavutil/hwcontext_vulkan.c b/media/ffvpx/libavutil/hwcontext_vulkan.c
--- a/media/ffvpx/libavutil/hwcontext_vulkan.c
+++ b/media/ffvpx/libavutil/hwcontext_vulkan.c
@@ -1720,7 +1720,8 @@ static int setup_queue_families(AVHWDeviceContext *ctx, VkDeviceCreateInfo *cd)
             if (p->limit_queues ||                                       \
                 p->dprops.driverID == VK_DRIVER_ID_NVIDIA_PROPRIETARY) { \
                 int max = p->limit_queues;                               \
-                if (type == VK_QUEUE_GRAPHICS_BIT)                       \
+                if (type == VK_QUEUE_GRAPHICS_BIT ||                     \
+                    type == VK_QUEUE_COMPUTE_BIT)                        \
                     hwctx->qf[i].num = FFMIN(hwctx->qf[i].num,           \
                                              max ? max : 1);             \
                 else if (max)                                            \
